src/components/ui/Hero/hero.scss

```diff
@@ -6,7 +6,7 @@
   width: 100%;
   background-color: var(--bg-secondary-default);
 
-  @include media(">=notebook") {
+  @include media(">=tablet") {
```

Comment:
I have an idea to fix this, slack me!
<img src="https://user-images.githubusercontent.com/11613011/149336852-6dcc5bf3-62a4-4991-98e7-b34c2537dc41.png" width="400" />